provider/database.py

```diff
@@ -18,13 +18,19 @@
 from cloudant.result import Result
 
 class Database:
+    dbname = "ow_kafka_triggers"
```

Comment:
Starting to the database name should be passed in as a parameter to the Docker run command instead of being hardcoded.